draft-ietf-core-target-attr.md

```diff
@@ -165,29 +168,38 @@ Reference:
-| ct              | Content-Format hint                                                 | IESG              | {{Section 7.2.1 of RFC7252}} |
-| obs             | observable resource                                                 | IESG              | {{Section 6 of RFC7641}}     |
-| hct             | HTTP-CoAP URI mapping template                                      | IESG              | {{Section 5.5 of RFC8075}}   |
-| osc             | hint: resource only accessible using OSCORE                         | IESG              | {{Section 9 of RFC8613}}     |
+| Attribute Name | Brief description                              | Change Controller | Reference                  |
+| href           | reserved (not useful as target attribute name) | IESG              | {{RFC6690}}                  |
+| anchor         | reserved (not useful as target attribute name) | IESG              | {{RFC6690}}                  |
+| rel            | reserved (not useful as target attribute name) | IESG              | {{RFC6690}}                  |
+| rev            | reserved (not useful as target attribute name) | IESG              | {{RFC6690}}                  |
+| hreflang       | (Web Linking)                                  | IESG              | {{-linking}}                 |
+| media          | (Web Linking)                                  | IESG              | {{-linking}}                 |
+| title          | (Web Linking)                                  | IESG              | {{-linking}}                 |
+| type           | (Web Linking)                                  | IESG              | {{-linking}}                 |
+| rt             | resource type                                  | IESG              | {{Section 3.1 of RFC6690}}   |
+| if             | interface description                          | IESG              | {{Section 3.2 of RFC6690}}   |
+| sz             | maximum size estimate                          | IESG              | {{Section 3.3 of RFC6690}}   |
+| ct             | Content-Format hint                            | IESG              | {{Section 7.2.1 of RFC7252}} |
+| obs            | observable resource                            | IESG              | {{Section 6 of RFC7641}}     |
+| hct            | HTTP-CoAP URI mapping template                 | IESG              | {{Section 5.5 of RFC8075}}   |
+| osc            | hint: resource only accessible using OSCORE    | IESG              | {{Section 9 of RFC8613}}     |
+| ep             | Endpoint Name                                  | IESG              | {{Section 9.3 of RFC9176}}   |
+| d              | Sector                                         | IESG              | {{Section 9.3 of RFC9176}}   |
+| base           | Registration Base URI                          | IESG              | {{Section 9.3 of RFC9176}}   |
+| et             | Endpoint Type                                  | IESG              | {{Section 9.3 of RFC9176}}   |
```

Comment:
These are limited in their use to endpoints in an RD; maybe introduce a means for indicating preconditions (eg. `rt="core.rd-ep"` here). That doesn't mean we have to allow overlapping use.